docs/governance/politeia.md

```diff
@@ -39,8 +39,8 @@ For now, the process is something like this:
 3. Valid proposals appear publicly on Politeia, open for discussion, but voting does not begin immediately.  	
 4. Ticket-voting interval (~1 week) can be triggered by the proposal owner. Until ticket-voting is triggered, the proposal can be edited by its owner in response to community feedback.
 5. Ticket-voting starts, ticket-holders vote through their wallet (details tbc).
-6. When the ticket-voting period ends, the proposal is formally approved or rejected. Some types of proposal will require a super-majority. Details like supermajorities and quorum required for different kinds of proposal are to be determined. 
-7. When a proposal with a budget and deliverables is approved, work can 	begin. The proposal owner can submit claims against that budget as deliverables are completed.
+6. When the ticket-voting period ends, the proposal is formally approved or rejected. A typical proposal must meet a 20% quorum and 60% approval threshold to pass. Some types of proposal will require a super-majority. Details like supermajorities and quorum required for different kinds of proposal are to be determined. 
```

Comment:
Should remove the previous placeholder text: "Some types of proposal will require a super-majority. Details like supermajorities and quorum required for different kinds of proposal are to be determined. " They've been determined now, that's what the new sentence is about :)

Reply:
Could add a different note about the potential for supermajority requirement to scale with proposal scale/importance in future, but I don't think this is the right place.

Reply:
Going with some copy I got from the Decred LinkedIn announcement of Pi, which I think is concise and adds more useful info:

"When the ticket-voting period ends, the proposal is formally approved or rejected. _There is a quorum requirement of 20% tickets voting Yes/No. For a proposal to pass it must achieve 60% Yes votes._"

I agree that a statement about the requirements scaling with importance in the future is out of place. It could easily get misinterpreted, or go out of date accidentally.